docs/deploy-cephfs.md

```diff
@@ -82,8 +82,8 @@ is used to define in which namespace you want the configmaps to be stored
 | `fsName`                                                                                            | yes            | CephFS filesystem name into which the volume shall be created                                                                                                                                                           |
 | `mounter`                                                                                           | no             | Mount method to be used for this volume. Available options are `kernel` for Ceph kernel client and `fuse` for Ceph FUSE driver. Defaults to "default mounter".                                                          |
 | `pool`                                                                                              | no             | Ceph pool into which volume data shall be stored                                                                                                                                                                        |
-| `volumeNamePrefix`                                                                                  | no             | Prefix to use for naming volumes (defaults to `csi-vol-`).                                                                                                                                                    |
-| `snapshotNamePrefix`                                                                                | no             | Prefix to use for naming snapshots (defaults to `csi-snap-`)
+| `volumeNamePrefix`                                                                                  | no             | Prefix to use for naming volumes (defaults to `csi-vol-`).                                                                                                                                                              |
```

Comment:
Unrelated change?

Reply:
this should not be an issue. I just opened and viewed `md` file.  the indentation got changes as the above line length is increased

Reply:
it should not be an issue, but it is bad practise to include changes that are not related to the intention of a PR